src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -101,7 +107,7 @@ CAC.Map.Control = (function ($, L, _) {
             deferred.fail(function(){return 'ERROR(' + error.code + '): ' + error.message; });
         };
 
-        if ('geolocation' in navigator) {
+        if (navigator.geolocation) {
```

Comment:
Good to know.  I'd pulled that from [the previous working branch](e3bbebb41e019ff0567ea23d9e7d1b652d151dce) and assumed it was necessary.  I'll revert that.